lib/modules/dosomething/dosomething_reportback/dosomething_reportback.admin.inc

```diff
@@ -238,12 +238,20 @@ function dosomething_reportback_files_form($form, &$form_state, $status = NULL,
  * Form submit callback for dosomething_reportback_files_form_submit.
  */
 function dosomething_reportback_files_form_submit($form, &$form_state) {
+  $user = dosomething_helpers_get_current_user();
+
   $rb_files = $form_state['values']['rb_files'];
   if (empty($rb_files) || !is_array($rb_files)) {
     return;
   }
 
   foreach ($rb_files as $fid => $values) {
+    $kudos = [
+      'fid' => $fid,
+      'uid' => $user->uid,
+      'tid' => dosomething_kudos_get_term_id_by_name(array_pop(Kudos::getApprovedReactions())),
```

Comment:
in theory... if we have multiple kudo types, this will just grab the first thing in the array..which might not always be a heart. maybe add a note for that in the function? or here?